sys/targets/targets.go

```diff
@@ -447,10 +465,17 @@ func initTarget(target *Target, OS, arch string) {
 		// https://github.com/google/syzkaller/pull/619
 		// https://github.com/google/syzkaller/issues/387
 		// https://github.com/google/syzkaller/commit/06db3cec94c54e1cf720cdd5db72761514569d56
-		target.CCompilerPrefix = ""
+		target.Triple = ""
+	}
+	if OS == "linux" && target.Triple != "" {
```

Comment:
It seems that `target.Triple != ""` condition should not affect `-ferror-limit=0` part below.

Reply:
Hopefully clang is not as broken as gcc and we don't need the check for `target.Triple != ""` at all.